ansible/roles/products/common/docker-container/tasks/modify-container.yml

```diff
@@ -23,9 +23,24 @@
   become: yes
   changed_when: false
 
-- name: Please wait and don't close the empty terminator window while aurora is executing the modify container script with these commands {{ modify_script_commands }}
+- name: Set use_retries to false if unspecified
+  ansible.builtin.set_fact:
+    use_retries: "{{ use_retries | default('false') }}"
+
+- name: Please wait and don't close the empty terminator window while aurora is executing the modify container script with these commands {{ modify_script_commands }} without retries
+  shell: "/tmp/{{ modify_script_name }}"
+  become: yes
+  changed_when: false
+  register: result
+  when: not skip_molecule_task|bool and not use_retries|bool
+
+- name: Please wait and don't close the empty terminator window while aurora is executing the modify container script with these commands {{ modify_script_commands }} with retries
   shell: "/tmp/{{ modify_script_name }}"
   become: yes
   changed_when: false
-  when: not skip_molecule_task|bool
-  
+  retries: 10
+  delay: 3
+  register: result
+
```

Comment:
Do we want this blank line?

Reply:
ff43939bc2e22f75aef883c93c5ac1c78bbb0a25